### DISCUSSION TITLE
Was für eine tolle Mod

### DISCUSSION THREAD

**Discussion by Michael170978** [Mogli12/AIVehicleExtension#261] [General]
Hallo Mogli

Habe rein zufällig diese Mod gefunden ich muss dir sagen ich bin begeistert!!!

Was für eine toller Helfer. Und ich Depp hab mich die ganze Zeit mit Courseplay gequält.

Ich möchte mich wirklich bedanken bei dir.

**Comment by Mogli12** [comment]
Vielen Dank!

Ich benutze übrigens auch Courseplay. Manchen Sachen gehen damit einfacher, manches gemt in AIVE besser.

**Comment by RalfGorden** [comment]
hallo wirklich guter Mod ich spiele mit dem AIVE  ,CP und AD , ergänzen sich alle 3
